README.md

```diff
@@ -46,101 +46,50 @@ This repository contains the ape-backend codebase which serves the Coordinape AP
 
 This is a project written in PHP using the [Laravel](https://laravel.com/) PHP framework. The database is mysql. Docker is used to setup the environment.
 
+--- 
 
-## Getting Started
-Welcome! We're so glad you're reading this and want to contribute! This section will get your dev environment setup. 
+## Start in [Coordinape](https://github.com/coordinape/coordinape)
 
-This app uses Docker.
+We are migrating from Laravel to Hasura GraphQL.
 
- 1.  Install Docker: https://docs.docker.com/engine/install/
+See https://github.com/coordinape/coordinape
+Where Laravel is included as a submodule and the docker-compose runs this as well as the other services.
 
- Once docker is installed and and the docker daemon is running you can check your docker version:
+# Legacy setup 
 
- ```bash
- > docker --version
-Docker version 20.10.7, build f0df350
- ```
+ 1. Install Docker: https://docs.docker.com/engine/install/
 
- 2. Setup your `.env` file by copying the example environment file:
+ 2. To Initialze the db standalone from hasura, uncomment
+`php artisan migrate` in `./services/start.sh`.
 
- ```
- cp .env.example .env
- ```
+ 3. `cp .env.example .env`
 
+ 4. `docker-compose up -d`
 
- 3. Use docker compose to launch the three containers required for the app to run:
- ```bash
-cd coordinape-backend
-docker compose up
-```
-
-This can take a while to build the Dockerfile and install all the app dependencies into `vendor/`.
-
-4. Now you need to populate your local database with some data.
+#### Populate your local db
 
 * **Option 0:** create data through console command.
 ```bash
 docker exec app php artisan ape:quickstart youraddresshere
 ```
+
 * **Option 1:** create data in the database directly.
     1. Create a protocol
     2. Create a circle - link to protocol_id
     3. Create a user - link circle_id to circle, address=your_address,role = 1 (for admin access)
     4. Create profile - just need an address
 
-* **Option 2:** restore database from a staging db dump (get from another dev)
-```bash
-docker exec -i mysql mysql -uroot -psecret laravel < database/db_dumps/staging_db_dump.sql
-```
-
-## Connect to database.
```

Comment:
I'd think we should move these debugging and DB connection notes somewhere rather than delete. Are this moved to the other repo's README?

Reply:
I am about to add notes to the other README, let me know what you feel is needed there..

For these, fwiw, mysql is no longer used, so these are out dated.